src/sdg_hub/blocks/llmblock.py

```
@@ -38,19 +53,43 @@ def server_supports_batched(client, model_id: str) -> bool:
+    parser_kwargs : Dict[str, Any], optional
+        Keyword arguments for the parser, by default {}.
+    model_prompt : str, optional
+        Template string for model prompt, by default "{prompt}".
+    model_id : Optional[str], optional
+        Model ID to use, by default None.
+    **batch_kwargs : Dict[str, Any]
+        Additional keyword arguments for batch processing.
+    """
+
     # pylint: disable=too-many-instance-attributes
     def __init__(
```

Pylint (3.3.7)

[warning] Dangerous default value {} as argument

(W0102)